modules/edge-identity/src/identity.rs

```diff
@@ -225,11 +240,14 @@ impl<T: Trait> Module<T> {
 		let id_type = record.identity_type.encode().clone();
 		let id = record.identity.encode().clone();
 		if approve {
+			// Add the registration bond amount on behalf of a successful verification
+			T::Currency::deposit_into_existing(&record.account, Self::registration_bond()).unwrap();
```

Comment:
If you use reserve, then you can change this into unreserve() like here https://github.com/paritytech/substrate/blob/master/srml/council/src/seats.rs#L523